### DISCUSSION TITLE
有人知道站点代理是什么原理吗

### DISCUSSION THREAD

**Discussion by Yuhanlll** [jxxghp/MoviePilot#1948] [讨论]
我设置了PROXY_HOST这个环境变量，确实可以代理连接github和tmdb，但是访问不了的pt站比如u2，勾选了代理还是没法连接，不知道这个地方点击代理后是不是用的PROXY_HOST里填的sockt5的代理，希望有人能帮我解惑，谢谢～

**Comment by junyuyuan** [comment]
我也遇到这个问题了，请问解决了么？

**Comment by cng10001** [comment]
是的，默认应该是填了代理的环境变量不对站点直接生效的，如果要让站点走代理，就要手动在站点管理里边把该站点的代理选项打开。
